src/libsync/theme.h

```diff
@@ -349,14 +349,20 @@ class OWNCLOUDSYNC_EXPORT Theme : public QObject
      */
     virtual QString versionSwitchOutput() const;
 
-
     /**
      * @brief Whether to show the option to create folders using "virtual files".
      *
+     * By default, this is the same as enableExperimentalFreatures()
+     */
+    virtual bool showVirtualFilesOption() const;
+
+    /**
+     * @brief Whether to show options considered as experimental.
+     *
      * By default, the options are not shown unless experimental options are
      * manually enabled in the configuration file.
      */
-    virtual bool showVirtualFilesOption() const;
+    virtual bool enableExperimentalFreatures() const;
```

Comment:
Its a Freature. But its consistently a Freature. You might want to fix the speling there.

Reply:
🙈